packages/prover/src/utils/rpc_provider.ts

```diff
@@ -34,12 +34,23 @@ export class ELRpc {
     this.logger = logger;
   }
 
+  /**
+   * Request the EL RPC Provider
+   *
+   * @template K
+   * @template E
+   * @param {K} method - RPC Method
+   * @param {ELApiParams[K]} params - RPC Params
+   * @param {{raiseError?: E}} [opts]
+   * @return {*}  {Promise<E extends false ? JsonRpcResponse<ELApiReturn[K]> : JsonRpcResponseWithResultPayload<ELApiReturn[K]>>}
+   * @memberof ELRpc
+   */
   async request<K extends keyof ELApi, E extends boolean>(
     method: K,
     params: ELApiParams[K],
-    opts: {raiseError: E}
+    opts?: {raiseError?: E}
```

Comment:
This can't be done because E is a generic. 

>  Type 'boolean' is not assignable to type 'E'. @lodestar/prover:   'boolean' is assignable to the constraint of type 'E', but 'E' could be instantiated with a different subtype of constraint 'boolean'.